### PR TITLE
Improved discussion thread empty state

### PR DESCRIPTION
**Pull request by BartoszJarocki**
Related issue: https://github.com/ParabolInc/parabol/issues/4813 

This is how it looks like on a big screen:
Screenshot 2021-07-20 at 17 34 03

This is how it looks like on a smaller screen - the empty state along with the header scrolls. I'm not sure if that's the best solution so if someone has better ideas I'd love to improve! 
Screenshot 2021-07-20 at 17 34 28

### CODE REVIEW

File: packages/client/components/DiscussionThreadListEmptyState.tsx [hunk 1]
```diff
@@ -23,10 +40,23 @@ const DiscussionThreadListEmptyState = (props: Props) => {
...
+    <DiscussionThreadEmptyStateRoot>
+      <EmptyDiscussionContainer>
+        <EmptyDiscussionSVG />
+      </EmptyDiscussionContainer>
+      <Message>{isReadOnly ? readOnlyMessage : message}</Message>
+      {!isReadOnly && (
+        <TangledArrowContainer>
+          <TangledArrowSVG />
+        </TangledArrowContainer>
+      )}
```
Comment (BartoszJarocki) [inline packages/client/components/DiscussionThreadListEmptyState.tsx:57]: It felt weird to have this arrow pointing to disabled input so I removed it in this case. Is that acceptable @enriquesanchez?

Comment (enriquesanchez) [reply packages/client/components/DiscussionThreadListEmptyState.tsx:57]: Oh, interesting! In what scenarios would this happen?

**Comment by enriquesanchez** [comment]
This is looking great, @BartoszJarocki 🙌

>This is how it looks like on a smaller screen - the empty state along with the header scrolls. I'm not sure if that's the best solution so if someone has better ideas I'd love to improve!

I think in this scenario we can remove the `TangledArrowSVG`. This way we keep the illustration and the text, and shouldn't create any unnecessary scrolling.

**Comment by BartoszJarocki** [comment]
@enriquesanchez Thank you for the review! I've added hiding arrow on smaller devices. 
Screenshot 2021-07-21 at 15 15 08

**Comment by mattkrick** [comment]
nice attention to detail, LGTM!

**Pull request merged**